src/components/networkComponent.ts

```diff
@@ -85,12 +85,14 @@ export default class {
 			this.statusBarItem.show();
 		});
 	}
-	onExtensionDeactivated() {
-		this.serverInstance.close();
-		this.statusBarItem.dispose();
+	// This will now be handled by the controlComponent.
```

Comment:
At what point is it handled? If this implementation is not yet finished then please do not push this PR

Reply:
It is handled in the `instance.ts` file.